configuration/initialization.sh

```diff
@@ -1,4 +1,82 @@
 #!/bin/bash
+
+# Default values for the variables
+PORT_DRAYAGE_ENABLED_DEFAULT="TRUE"
+INFRASTRUCTURE_ID_DEFAULT="rsu_1234"
+INFRASTRUCTURE_NAME_DEFAULT="East Intersection"
+V2XHUB_IP_DEFAULT="127.0.0.1"
+SIMULATION_MODE_DEFAULT="FALSE"
+SIMULATION_IP_DEFAULT="127.0.0.1"
+SENSOR_JSON_FILE_PATH_DEFAULT="/var/www/plugins/MAP/sensors.json"
+
+echo "Setting up the environment..."
+
+# Repository URL
+repo_url_latest="https://api.github.com/repos/usdot-fhwa-OPS/V2X-Hub/releases/latest"
+
+# Getting the latest release information using curl
+release_info=$(curl -sSL $repo_url_latest)
+
+# Parsing the JSON response to get the tag_name (version) of the latest release
+latest_version=$(echo "$release_info" | grep -o '"tag_name": *"[^"]*"' | cut -d '"' -f 4)
+
+# Fetching all tags from Git repository
+tags=$(git ls-remote --tags https://github.com/usdot-fhwa-OPS/V2X-Hub.git | awk -F/ '{print $3}' | sort -V)
+
+# Displaying all available versions
+echo "Available versions:"
+echo "$tags"
+
+# select a version or accept the latest version as default
+read -r -p "Enter V2X-Hub Version (choose from the above, or press Enter to use latest version $latest_version): " chosen_version
+V2XHUB_VERSION=${chosen_version:-$latest_version}
+
+# Enable Port Drayage functionality
+read -r -p "Enable Port Drayage functionality (TRUE/FALSE, or press Enter to use default as $PORT_DRAYAGE_ENABLED_DEFAULT): " PORT_DRAYAGE_ENABLED
+PORT_DRAYAGE_ENABLED=${PORT_DRAYAGE_ENABLED:-$PORT_DRAYAGE_ENABLED_DEFAULT}
+
+# Infrastructure id
+read -r -p "Enter Infrastructure id (or press Enter to use default as $INFRASTRUCTURE_ID_DEFAULT): " INFRASTRUCTURE_ID
+INFRASTRUCTURE_ID=${INFRASTRUCTURE_ID:-$INFRASTRUCTURE_ID_DEFAULT}
+
+# Infrastructure name
+read -r -p "Enter Infrastructure name (or press Enter to use default as $INFRASTRUCTURE_NAME_DEFAULT): " INFRASTRUCTURE_NAME
+INFRASTRUCTURE_NAME=${INFRASTRUCTURE_NAME:-$INFRASTRUCTURE_NAME_DEFAULT}
+
+# V2XHub IP
+read -r -p "Enter V2XHub IP (or press Enter to use default as $V2XHUB_IP_DEFAULT): " V2XHUB_IP
+V2XHUB_IP=${V2XHUB_IP:-$V2XHUB_IP_DEFAULT}
+
+# Simulation Mode
+read -r -p "Simulation Mode (TRUE/FALSE, or press Enter to use default as $SIMULATION_MODE_DEFAULT): " SIMULATION_MODE
+SIMULATION_MODE=${SIMULATION_MODE:-$SIMULATION_MODE_DEFAULT}
+
+# In Simulation Mode
+if [[ $SIMULATION_MODE == "TRUE" ]]; then
+    # Simulation IP
+    read -r -p "Enter Simulation IP (or press Enter to use default is $SIMULATION_IP_DEFAULT): " SIMULATION_IP
+    SIMULATION_IP=${SIMULATION_IP:-$SIMULATION_IP_DEFAULT}
+
+    # Sensor Configuration File Path
+    read -r -p "Enter Sensor Configuration File Path (or press Enter to use default as $SENSOR_JSON_FILE_PATH_DEFAULT): " SENSOR_JSON_FILE_PATH
+    SENSOR_JSON_FILE_PATH=${SENSOR_JSON_FILE_PATH:-$SENSOR_JSON_FILE_PATH_DEFAULT}
+fi
+
+# Write to .env file
```

Comment:
If .env file already exists we may want to warn user that we are overwriting of even ask for permission to overwrite

Reply:
Updated the script to ask for permission to overwrite the existing .env file